hpx-sys/include/wrapper.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <hpx/hpx_init.hpp>
+#include <iostream>
+#include <cstdint>
+#include <vector>
+
+#include "rust/cxx.h"
+
+inline std::int32_t start(rust::Fn<int(int, char **)> rust_fn, int argc,
+                          char **argv) {
+  return hpx::init(
```

Comment:
Should we keep it `start` or `init` ? For the sake of consistency?

Reply:
using `init` instead of `start` for this 'coz currently depending on `finalize, disconnect, suspend` to close hpx runtime ensuring  consistency with [hpx-docs](https://hpx-docs.stellar-group.org/branches/master/html/api/public_api.html#hpx-init-hpp)